flight_planning/v1/flight_planning.yaml

```diff
@@ -116,60 +132,91 @@ components:
       type: object
       required:
-        - intended_flight
+        - flight
         - request_id
       properties:
-        intended_flight:
+        flight_plan:
           $ref: '#/components/schemas/FlightPlan'
         request_id:
+          type: string
           description: >-
             ID uniquely identifying the upsertion request.  If additional requests are received with the same
             request_id, the response from the first request should be returned, or an error indicated.
-          anyOf:
-            - $ref: './astm_f3548_21.yaml#/components/schemas/UUIDv4Format'
+
+    PlanningActivityResult:
+      type: string
+      description: >-
+        The result of a flight planning activity.
+
+          - `Completed`: The user's flight plan has been updated according to the situation specified by the user. 
+
+          - `Rejected`: The updates the user requested to their flight plan are not allowed according to the rules under which the flight plan is being managed.  The reasons for rejection may include a disallowed conflict with another flight during preflight.
+        
+          - `Failed`: The USS was not able to successfully authorize or update the flight plan due to a problem with the USS or a downstream system.
+
+          - `NotSupported`: The USS's implementation does not support the attempted interaction.  For instance, if the request specified a high-priority flight and the USS does not support management of high-priority flights.
+      enum: [Completed, Rejected, Failed, NotSupported]
+      example: Completed
+
+    FlightPlanStatus:
+      type: string
+      description: >-
+        The status of the user's flight plan.
+
+          - `NotPlanned`: The USS has not created an authorized flight plan for the user.
+
+          - `Planned`: The USS has created an authorized flight plan for the user, but the user may not yet start flying (even if within the time bounds of the flight plan).
+
+          - `ReadyToFly`: The flight plan is ready for the operator to nominally fly within the bounds (including time) of the flight plan.
+
+          - `OffNominal`: The flight plan now reflects the operator's actions, but the flight plan is not in a nominal state (e.g., the USS has placed the ASTM F3548-21 operational intent into one of the Nonconforming or Contingent states).
+
+          - `Closed`: The flight plan was closed successfully by the USS and is now out of the UTM system.
+      enum: [NotPlanned, Planned, ReadyToFly, OffNominal, Closed]
+      example: Planned
+
+    AdvisoryInclusion:
+      type: string
+      description: >-
+        Indication of whether any advisories or conditions were provided to the user along with the result of an
+        associated flight planning attempt.
+
+          - `Unknown`: It is unknown or irrelevant whether advisories or conditions were provided to the user
```

Comment:
nit: any reason why including `Unknown` and not just leaving the field empty in this case?

Reply:
It's in an effort to be as accommodating to different tooling as possible.  For instance, the problem-child of RPC tooling is Google's Protocol Buffers (protobuf) which is great when designing APIs using protobufs, but is not accommodating for many non-protobuf API design patterns.  One specific proto3 "characteristic" (shortcoming? oversight? questionable design choice?) is that it is difficult to differentiate between default values and "missing", and it's difficult to actuate "omit" versus "default value".  So, to maximize compatibility, a good API design practice is to specify an explicit default value that has the same effect as omitting the field.  That way, tooling that prefers omitting values can do so, and tooling that prefers explicit values for all "primitive" (non-object) types can do so.

Reply:
Makes sense, thanks for the clarification :+1: